### PR TITLE
Terms and Conditions

### PR DESCRIPTION
**Pull request by MatkoTom**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant
motivation and context. List any dependencies that are required for this change.

Implemented terms and conditions to the beginning of the onboarding process and in user settings. 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can
reproduce. Please also list any relevant details for your test configuration. All testing was done
manually by following, plainly, functionality requirements. If there are some special edge cases or
special ways that things need to be tested they will be mentioned below.

Dev testing

**Test Configuration**:

* Firmware version: G980FXXSFFVHA
* Hardware: SAMSUNG Galaxy S20
* SDK: Android 13

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules

tempFile1689157074489
tempFile1689157074646
tempFile1689157114845
tempFile1689157121081

### CODE REVIEW

File: app/src/main/java/com/clover/studio/spikamessenger/ui/onboarding/terms_and_conditions/TermsAndConditions.kt [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+            ForegroundColorSpan(
+                resources.getColor(
+                    R.color.primary_color,
+                    null
+                )
+            ), 35, binding.tvWelcomeDescription.text.length, Spannable.SPAN_EXCLUSIVE_EXCLUSIVE
+        )
+        spannable.setSpan(
+            clickableSpan,
+            35,
```
Comment (ivolarevic) [inline app/src/main/java/com/clover/studio/spikamessenger/ui/onboarding/terms_and_conditions/TermsAndConditions.kt:55]: What is this 35? Length of words that must be other color? Maybe you can extract it to some const var for better reading?

**Review by ivolarevic — DISMISSED** [review]
LGTM

**Review by ivolarevic — APPROVED** [review]
LGTM
